{"event": "verbose", "uuid": "822d4b4e-73fa-4189-abf1-099645cd9365", "counter": 146, "stdout": "\u001b[0;31mERROR! Invalid extra vars data supplied. '@/Users/devops/data/project/python/pgoops/upload/iac/test2/env/extravars' could not be made into a dictionary\u001b[0m", "start_line": 145, "end_line": 146, "runner_ident": "50caa126-7c4d-474b-9711-0a353c8b242c", "created": "2022-04-21T15:33:11.030188"}